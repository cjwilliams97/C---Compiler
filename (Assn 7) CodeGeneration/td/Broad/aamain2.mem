Func: main returns type void [mem: Global loc: 0 size: -2] [line: 1]
.   Child: 1 Compound [mem: None loc: 0 size: -2] [line: 2]
.   .   Child: 1 Call: output of type void [line: 3]
.   .   .   Child: 0 Const 666 of type int [line: 3]
.   .   Sibling: 1 Call: outputb of type void [line: 4]
.   .   .   Child: 0 Const true of type bool [line: 4]
.   .   Sibling: 2 Call: outnl of type void [line: 5]
.   .   Sibling: 3 Call: input of type int [line: 6]
.   .   Sibling: 4 Call: inputb of type bool [line: 7]
.   .   Sibling: 5 Return [line: 9]
Offset for end of global space: 0
Number of warnings: 0
Number of errors: 0
