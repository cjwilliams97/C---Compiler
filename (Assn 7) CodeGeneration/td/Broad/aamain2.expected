Loading file: aamain2.tm
666 T
entered: 496
entered: T
Bye.
